README.md

```diff
@@ -7,19 +7,24 @@ should work on all Debian and RHEL platform family OS's.
 Utilization
 ------------
 
-This cookbook works as a wrapper around the community MySQL cookbook(version 5.x only)
-to allow for the creation of master/slave and master/multi-slave MySQL systems.
+This cookbook provides libraries to work along with MySQL community recipe to
```

Comment:
Could we document those resources ? With examples and a list of attributes/actions ?